### PR TITLE
Update the link for permissions

### PR DESCRIPTION
**Pull request by shavidissa**

### CODE REVIEW

File: csharp/wavefrontsender.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ IWavefrontSender wavefrontSender = wfProxyClientBuilder.Build();
 When sending data via direct ingestion, you need to create a `WavefrontDirectIngestionClient`, and build it with the Wavefront URL and API token to send data directly to Wavefront.
 
 >**Prerequisite**
-> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-and-permissions).
+> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
```
Comment (rkempf42) [inline csharp/wavefrontsender.md]: Title of the section is now "Examine Groups, Roles, and Permissions"

**Review by rkempf42 — CHANGES_REQUESTED** [review]
Please change the title of the section in all 4 places. Thanks!

File: csharp/wavefrontsender.md (hunk 1 above)
Comment (shavidissa) [reply csharp/wavefrontsender.md]: Thank you! Updated the title.

**Review by rkempf42 — APPROVED** [review]
LGTM!

**Pull request merged**